### PR TITLE
Add/content UI

### PR DESCRIPTION
**Pull request by HassanKhan8264**

### CODE REVIEW

File: backend/models/schema.js [hunk 1]
```diff
@@ -15,6 +15,9 @@ const userInfoSchema = new Schema({
     message: {
         type: Types.String, Required: false,
     },
+    selectedExamControl: {
```
Comment (HassanKhan9264) [inline backend/models/schema.js:18]: what does this do

File: client/src/app/shared/components/user-form/user-form.component.html [hunk 2]
```diff
@@ -7,11 +7,11 @@ <h1 class="heading font-bold">
       {{data.formHeading ?data.formHeading : formHeading}}
     </h1>
   </div>
-  <form #formDirective="ngForm"  [formGroup]="userInfoForm" (ngSubmit)="onSubmit(formDirective)">
+  <form #formDirective="ngForm" [formGroup]="userInfoForm" (ngSubmit)="onSubmit(formDirective)">
     <div class="input-wrapper">
       <mat-form-field appearance="fill" [ngStyle]="{'width': data.inputWidth? '500px' : '100%'}">
         <mat-label>Name</mat-label>
-        <input matInput formControlName="name" placeholder="Ex:Hassan kahn" />
+        <input matInput formControlName="name" placeholder="Ex:Jhon Doe" />
```
Comment (HassanKhan9264) [inline client/src/app/shared/components/user-form/user-form.component.html:14]: change JHon spelling

**Review by HassanKhan9264 — CHANGES_REQUESTED** [review]

File: backend/models/schema.js (hunk 1 above)
Comment (HassanKhan8264) [reply backend/models/schema.js:18]: this add exam in db

File: client/src/app/shared/components/user-form/user-form.component.html (hunk 2 above)
Comment (HassanKhan8264) [reply client/src/app/shared/components/user-form/user-form.component.html:14]: ok i'll do that

File: client/src/app/layout/components/footer/footer.component.html [hunk 3]
```diff
@@ -2,23 +2,23 @@
...
       <div class="row">
-        <div class="col-md-2 col-lg-2 col-xl-2 mx-auto mb-4">
+        <!-- <div class="col-md-2 col-lg-2 col-xl-2 mx-auto mb-4">
           <img
             src="assets/images/academics.jpg"
             alt="Company Logo"
             class="img-fluid"
           />
-        </div>
+        </div> -->
```
Comment (HassanKhan8264) [inline client/src/app/layout/components/footer/footer.component.html:11]: xsxsdsd

Comment (HassanKhan8264) [reply client/src/app/layout/components/footer/footer.component.html:11]:
```suggestion
        <!-- <div class="col-md-2 col-lg-2 col-xl-2 mx-auto mb-4">
          <img
            src="assets/images/academics.jpg"
            alt="Company Logo"
            class="img-fluid"
          />
        </div> -->
```

**Review by HassanKhan9264 — APPROVED** [review]

**Pull request merged**